docs/usage.md

```diff
@@ -3,130 +3,441 @@
 
-Once FortiAI starts generating the playbook steps, a loader indicates the progress. After playbook steps have been generated, a playbook block is suggested in the playbook designer.
+>[!NOTE]
+>Ensure that the information provided for updates is accurate and complete to maintain the integrity of the records and support your workflow.
 
-The FortiAI-generated playbook block may require review, which you can perform by clicking the playbook steps and examining each step. Following image shows the *Determine Indicator Reputation* step:
+## Block Indicator
 
-![Reviewing a playbook step generated by FortiAI](../docs/res/FortiAIGeneratedPb_StepReview.png)   
+Blocking an indicator within the FortiAI assistant involves using a configured connector to prevent or mitigate the impact of the indicator. This functionality is crucial for managing security threats by stopping potentially harmful indicators from affecting your systems.
 
-Post-review, you can connect the AI generated playbook blocks to the main playbook flow and create a complete workflow.
+### Use Case Example: Blocking a Malicious IP Address
 
-As  demonstrated, this represents a significant advancement in playbook design and is of enormous  assistance to individuals who are just starting out developing playbooks.
+A common use case is blocking a malicious IP address that has been identified as a threat. For example, if an IP address is flagged as malicious, blocking it prevents further interactions with the threat, protecting your network from potential harm.
 
-#### Common use cases for generating playbook steps using FortiAI
+### Blocking Indicators Using FortiAI Assistant
 
-Some other playbook generation use cases that you could use to familiarize yourself with FortiAI are:
+1. **Initiate Blocking Action**:
+   - Issue a command to block the indicator using the [configured connector]. For example:
 
-- Extract IOCs from CSV and Create the Indicator Record for each
-- Extract Indicators from Alert Source Data and Create Indicator Records for each
-- Fetch Unread Emails from Exchange and create Alert record
-- Notify using Slack when Alert severity changes
-- On Create of alert add a comment *Hello, Analyst!* on Alert
-- Check the IP's reputation using VirusTotal.
-    - If it's considered malicious, set the indicator reputation as malicious.
-    - If it's considered suspicious, set the indicator reputation as suspicious.
-    - Else, set the indicator reputation as good
-- Submit a file for analysis to FortiSandbox and obtain the verdict for the analysis job
-- On Delete of an alert, using Exchange send an email to Admin
-- Find the reputation of the indicator using VirusTotal and prompt the user for action: either block it using FortiGate or take no action.
-- Manually trigger a playbook for an Alert Record to change the status to Investigating.
+    >*Block IP address `[IP-ADDRESS]` using the [configured connector]*
```

Comment:
change the prompt to "">*Block IP address `[IP-ADDRESS]` *

Reply:
Please also add that Block indicator playbook will be triggered to block an IOC.
Note: If playbook takes time to execute then it will present you a playbook execution log as part of response so that user can navigate and check its status.

Reply:
Done